Composer/packages/ui-plugins/lg/src/LgField.tsx

```diff
@@ -48,19 +48,39 @@ const LgField: React.FC<FieldProps<string>> = props => {
     [lgName, lgFileId]
   );
 
-  const template = (lgFile &&
-    lgFile.templates &&
-    lgFile.templates.find(template => {
+  const hasTemplate = useMemo(() => lgFile?.templates.find(t => t.name === lgName), []);
+  const template = (hasTemplate &&
```

Comment:
Here seems no need to find again, if hasTemplate is not a boolean value. it's equal to
```
const template = hasTemplate || {
     name: lgName,
     parameters: [],
     body: getInitialTemplate(name, value),
   };
```